Vahag/databases/29-04-2024/Employee.sql

```diff
@@ -0,0 +1,38 @@
+use Hospital;
+
+--1
+select h.address as H_address, h.name as H_name, 
+d.name as D_name, doc.name as Doc_name, p.name as P_name 
+from Hospital h
+inner join Department d on h.id = d.hospital_id
+inner join Doctor doc on doc.department_id = d.id
+inner join Patient p on p.id = d.hospital_id
```

Comment:
`p.id = d.hospital_id` is an incorrect comparison, as they are not the same column

Reply:
Done:)